clients/privacy-center/cypress/e2e/consent-notices.cy.ts

```diff
@@ -313,7 +312,7 @@ describe("Privacy notice driven consent", () => {
         const { preferences } = body;
         expect(
           preferences.map((p: ConsentOptionCreate) => p.preference)
-        ).to.eql(["opt_in", "opt_in", "acknowledge"]);
+        ).to.eql(["opt_in", "opt_out", "acknowledge"]);
```

Comment:
changed the cookie to make "advertising" opt out so that we have a test case where we are setting a value to `false` as well 👍

Reply:
thank you!